client/web/src/enterprise/batches/list/BatchChangeListPage.tsx

```diff
@@ -160,10 +165,12 @@ export const BatchChangeListPage: React.FunctionComponent<BatchChangeListPagePro
                 <Container className="mb-4">
                     <ConnectionContainer>
                         <div className={styles.filtersRow}>
-                            {(licenseAndUsageInfo?.allBatchChanges.totalCount || 0) > 0 && (
-                                <h3 className="align-self-end flex-1">{lastTotalCount} batch changes</h3>
+                            {typeof currentTotalCount === 'number' && lastTotalCount !== undefined && (
```

Comment:
Maybe to be consistent:
```suggestion
                            {typeof currentTotalCount === 'number' && typeof lastTotalCount === 'number' && (
```